MdePkg/Test/Mock/Include/GoogleTest/Library/MockSafeIntLib.h

```diff
@@ -0,0 +1,1001 @@
+/** @file MockSafeIntLib.h

```

Comment:
edk2 repo does not put the file name in doxygen @file statements.  Removes the need to update that content if file is renamed.

Reply:
Done